tests/components/plex/test_media_search.py

```diff
@@ -57,6 +57,31 @@ async def test_media_lookups(
         )
     assert "Media for key 123 not found" in str(excinfo.value)
 
+    # Search with a different specified username
+    with (
+        patch(
+            "plexapi.library.LibrarySection.search",
+            __qualname__="search",
+        ) as search,
+        patch(
+            "plexapi.myplex.MyPlexAccount.user",
+            __qualname__="user",
+        ) as plex_account_user,
+    ):
```

Comment:
Ideally we patch where the library is used, so somewhere in `homeassistant.components.plex....`

What does qualname do?

Reply:
I’m following convention in the file here. This is the way the rest of the tests are patched.